csv_detective/explore_csv.py

```diff
@@ -353,6 +356,8 @@ def routine(
             f'Routine completed in {round(time() - start_routine, 3)}s',
             time() - start_routine
         )
+    if output_df:
+        return return_dict, table
```

Comment:
I would change the naming of "`table`" to "`df`"

Reply:
I think I'll make a PR just to rename all objects that have a clanky name now, it'll follow right after this one